### DISCUSSION TITLE
无法显示图片**Can't load Image**

### DISCUSSION THREAD

**Discussion by xwklib** [hexojs/hexo#4924] [Question / Answer]
[My Blog URL](https://wkxuan.cn/2022/03/28/Ubuntu%E5%AE%89%E8%A3%85Jenkins%E5%B9%B6%E8%87%AA%E5%8A%A8%E5%8C%96%E9%83%A8%E7%BD%B2Maven%E9%A1%B9%E7%9B%AE/#more)
只有新发布的页面没法显示图片
以前的页面的图片都是正常显示的

**Comment by uiolee** [comment]
Is this still an issue? I can't see your blog.
